src/network/netdev/netdev-gperf.gperf

```diff
@@ -234,6 +234,7 @@ Bridge.VLANFiltering,                     config_parse_tristate,
 Bridge.VLANProtocol,                      config_parse_vlanprotocol,                 0,                             offsetof(Bridge, vlan_protocol)
 Bridge.STP,                               config_parse_tristate,                     0,                             offsetof(Bridge, stp)
 Bridge.MulticastIGMPVersion,              config_parse_uint8,                        0,                             offsetof(Bridge, igmp_version)
+Bridge.FDBMaxLearned,                     config_parse_uint32,                       0,                             offsetof(Bridge, fdb_max_learned)
```

Comment:
Missing documentation for the setting. Please update man/systemd.netdev.xml .

Reply:
Ok. Added